kong/plugins/apifortress/fortress_serializer.lua

```diff
@@ -55,19 +55,20 @@ function _M.serialize(ngx)
   for k, v in pairs(ngx_req.get_headers()) do
     local val = v
```

Comment:
Why create another local variable? why not stick to v, or do `for name, value in pairs(...` or something similar if you want a clearer naming?